internal/scraper/testdata/ping.txt

```diff
@@ -1,14 +1,17 @@
 # HELP probe_dns_lookup_time_seconds Returns the time taken for probe dns lookup in seconds
 # TYPE probe_dns_lookup_time_seconds gauge
-probe_dns_lookup_time_seconds 1.4305e-05
+probe_dns_lookup_time_seconds 1.742e-06
```

Comment:
These changes are making this PR (and the git history) a bit more noisy. Do we need all of these changes, or could we omit them? It'd be great if we could reset these, and just keep the new additions on lines 12-14.

Reply:
I would rather not hand-edit the generated files.

Reply:
It's just a matter of selectively staging changes, no? `git add -p`? I think it's worth it, if you believe that code (and the git log) is read more times than it is written 🤔 .